build-system/completion/__init__.py

```diff
@@ -33,9 +34,9 @@ def install_bootloader ():             return 'bootloader'
 
 def setup ():
    if platform.system () == 'Darwin':
-                                       return 'setup', ZeroOrMore ([with_xcode_support])
+                                       return 'setup', ZeroOrMore ([with_xcode_support, with_vscode_support])
    else:
-                                       return 'setup'
+                                       return 'setup', ZeroOrMore ([with_vscode_support])
 def init ():                           return 'init', ZeroOrMore ([name, language])
 def configure ():                      return 'configure'
 def build ():                          return 'build', [build_simulator, build_firmware, build_hardware]
```

Comment:
Lower in `DESCRIPTION` the description for `--with-vscode-support` is missing.
Actually also the case for `--with-xcode-support`, so maybe fill free to put this in this PR as well as a typo correction.

```
'--with-xcode-support': 'for Xcode support',
'--with-vscode-support': 'for Visual Studio Code support',
```

Reply:
Will do 👍 , I was not sure if it was missing on purpose or not in the first place for xcode.